DPGAnalysis/HcalNanoAOD/python/customiseHcalCalib_cff.py

```diff
@@ -35,25 +35,7 @@ def customiseHcalCalib(process):
     process.raw2digi_step = cms.Path(process.hcalCalibDigiSequence, process.RawToDigiTask)
 
     # Insert the HLT filter at start of user path and nanoaod endpath
-    process.user_step.insert(0, process.hcalCalibHLTFilter)
+    process.nanoAOD_step.insert(0, process.hcalCalibHLTFilter)
```

Comment:
a safer way, IMO, would be to list of path of the process and insert the filter in first position, instead of calling explicitly `nanoAOD_step` and `NANOAODoutput_step` without checking their existence in the process

Reply:
It seems to work when I replace that line with:
```
process.paths.values()[0].insert(0, process.hcalCalibHLTFilter)
```
Does that seem reasonable to you? Or is there a more preferred way to do this? (I don't see this trick anywhere else in CMSSW with an LXR search...)

Reply:
something like 
```
for path in self.process.paths:
     path.insert(0,process.hcalCalibHLTFilter)
```
I think. and verify that it does prepend it to the output path too at the next line